pkg/sharing/shared.go

```diff
@@ -239,6 +244,39 @@ func UpdateShared(inst *instance.Instance, msg TrackMessage, evt TrackEvent) err
 	return couchdb.UpdateDoc(inst, &ref)
 }
 
+// RemoveReferences deletes the references containing the sharingid
```

Comment:
s/RemoveReferences/RemoveSharedRefs/

[nitpick] The "references"  are the referenced_by of the files in my mind, I'd prefer to avoid this term here